spark-hpcc/src/main/java/org/hpccsystems/spark/datasource/HpccOptions.java

```diff
@@ -140,6 +142,47 @@ public HpccOptions(TreeMap<String, String> parameters) throws Exception
         {
             spanID = (String) parameters.get("spanid");
         }
+
+        if (parameters.containsKey("fileParts"))
+        {
+            String filePartsStr = (String) parameters.get("fileParts");
+            String[] filePartsStrArray = filePartsStr.split(",");
+            for (String filePartStr : filePartsStrArray)
+            {
+                filePartStr = filePartStr.trim();
+                // Check if str is a range
+                if (filePartStr.contains("-"))
```

Comment:
IDK if it makes sense but if we only want to read a single part, do we set filePartStr to 1-1 ?

Reply:
Also, does this syntax follow some sort of standard, or is it a new protocol

Reply:
If you want a single file you could just do "1" it support ranges, comma delimited list of files, or combinations of both IE: "1,2,3,7-10". No real standard just what I came up with

Reply:
Great! let's document the full syntax in spark-hpcc/Examples/PySparkExample.ipynb